### PR TITLE
update rubygem-jwt scl package

### PR DESCRIPTION
**Pull request by timogoebel**
This is my packaging attempt on https://github.com/theforeman/foreman/pull/5596.

@evgeni, @ekohl: Mind taking a look?

### CODE REVIEW

**Comment by timogoebel** [comment]
As I have no idea what I'm doing, so feel free to just close this PR and start from scratch. ;-)

File: packages/foreman/tfm-rubygem-jwt/tfm-rubygem-jwt.spec [hunk 1]
```diff
@@ -0,0 +1,93 @@
+%{?scl:%scl_package rubygem-%{gem_name}}
+%{!?scl:%global pkg_name %{name}}
+
+%global gem_name jwt
+
+Summary: JSON Web Token implementation in Ruby
+Name: %{?scl_prefix}rubygem-%{gem_name}
```
Comment (evgeni) [inline packages/foreman/tfm-rubygem-jwt/tfm-rubygem-jwt.spec]: Can you explicitly write `tfm-rubygem` here please?

**Comment by evgeni** [comment]
See https://github.com/theforeman/foreman-packaging/blob/rpm/develop/packages/foreman/tfm-rubygem-rest-client/README ;)

**Comment by timogoebel** [comment]
Nice, I updated the spec to follow the gem2rpm style and tests passed. :-)

**Comment by evgeni** [comment]
Okay, so the diff does look sane to me, and it builds correctly. That's good.

However, we ship [tfm-rubygem-signet](https://github.com/theforeman/foreman-packaging/blob/rpm/develop/packages/foreman/rubygem-signet/rubygem-signet.spec#L24-L25), which does require `jwt` < 2.0:
https://github.com/theforeman/foreman-packaging/blob/a87d4b5f7c414d27ec76275595e91d3b7808a677/packages/foreman/rubygem-signet/rubygem-signet.spec#L24-L25

:sob: 

Let's see if we can/should upgrade that too?

:rabbit:

**Comment by timogoebel** [comment]
> However, we ship [tfm-rubygem-signet](https://github.com/theforeman/foreman-packaging/blob/rpm/develop/packages/foreman/rubygem-signet/rubygem-signet.spec#L24-L25), which does require `jwt` < 2.0:

That explains why the package was there in the first place.

> Let's see if we can/should upgrade that too?

Is that something you can take care of? 👼

**Comment by evgeni** [comment]
fwiw, if I re-generate the package using the current `gem2rpm` template, I end up with the attached diff, maybe worth incorporating it.

```diff
diff --git a/packages/foreman/tfm-rubygem-jwt/tfm-rubygem-jwt.spec b/packages/foreman/tfm-rubygem-jwt/tfm-rubygem-jwt.spec
index 48e877484..b742c1b55 100644
--- a/packages/foreman/tfm-rubygem-jwt/tfm-rubygem-jwt.spec
+++ b/packages/foreman/tfm-rubygem-jwt/tfm-rubygem-jwt.spec
@@ -1,40 +1,43 @@
+# Generated from jwt-2.1.0.gem by gem2rpm -*- rpm-spec -*-
+# template: scl
 %{?scl:%scl_package rubygem-%{gem_name}}
 %{!?scl:%global pkg_name %{name}}
 
 %global gem_name jwt
 
-Summary: JSON Web Token implementation in Ruby
 Name: tfm-rubygem-%{gem_name}
 Version: 2.1.0
 Release: 1%{?dist}
+Summary: JSON Web Token implementation in Ruby
 Group: Development/Languages
 License: MIT
-URL: https://github.com/progrium/ruby-jwt
+URL: https://github.com/jwt/ruby-jwt
 Source0: https://rubygems.org/gems/%{gem_name}-%{version}.gem
 
+# start specfile generated dependencies
 Requires: %{?scl_prefix_ruby}ruby(release)
-Requires: %{?scl_prefix_ruby}ruby(rubygems)
 Requires: %{?scl_prefix_ruby}ruby >= 2.1
-
+Requires: %{?scl_prefix_ruby}ruby(rubygems)
 BuildRequires: %{?scl_prefix_ruby}ruby(release)
-BuildRequires: %{?scl_prefix_ruby}rubygems-devel
 BuildRequires: %{?scl_prefix_ruby}ruby >= 2.1
+BuildRequires: %{?scl_prefix_ruby}rubygems-devel
 BuildArch: noarch
 Provides: %{?scl_prefix}rubygem(%{gem_name}) = %{version}
-%{?scl:Obsoletes: ruby193-rubygem-%{gem_name}}
+# end specfile generated dependencies
 
 %description
-A Ruby implementation of JSON Web Token draft 06.
+A pure ruby implementation of the RFC 7519 OAuth JSON Web Token (JWT)
+standard.
+
 
 %package doc
 Summary: Documentation for %{pkg_name}
 Group: Documentation
 Requires: %{?scl_prefix}%{pkg_name} = %{version}-%{release}
-%{?scl:Obsoletes: ruby193-rubygem-%{gem_name}-doc}
 BuildArch: noarch
 
 %description doc
-Documentation for %{pkg_name}
+Documentation for %{pkg_name}.
 
 %prep
 %{?scl:scl enable %{scl} - << \EOF}
@@ -75,7 +78,6 @@ cp -pa .%{gem_dir}/* \
 %license %{gem_instdir}/LICENSE
 %{gem_instdir}/Manifest
 %{gem_libdir}
-%exclude %{gem_instdir}/ruby-jwt.gemspec
 %exclude %{gem_cache}
 %{gem_spec}
 
@@ -86,6 +88,7 @@ cp -pa .%{gem_dir}/* \
 %{gem_instdir}/Gemfile
 %doc %{gem_instdir}/README.md
 %{gem_instdir}/Rakefile
+%{gem_instdir}/ruby-jwt.gemspec
 %{gem_instdir}/spec
 
 %changelog
```

**Comment by timogoebel** [comment]
Incorporated the changes.

**Comment by evgeni** [comment]
https://github.com/theforeman/foreman-packaging/pull/3467 bumps `rubygem-signet` to `0.11.0`, which requires `jwt >= 1.5 < 3`, so `2.1` will be fine :)

**Comment by evgeni** [comment]
I do wonder why this PR didn't raise a repoclosure error. As technically the *current* `signet` becomes uninstallable as soon this PR is merged.

**Comment by evgeni** [comment]
build: http://koji.katello.org/koji/taskinfo?taskID=174966

**Pull request merged**